src/BeboerWeb.Mvc/Views/Home/Index.cshtml

```diff
@@ -5,12 +5,12 @@
 <div class="text-center">
     <h1 class="welcome-header">Velkommen til BeboerWeb</h1>
     @await Html.PartialAsync("Partials/MenuList", new List<MenuItem> {
-        new MenuItem { Icon = "bi bi-building",  MenuTitle = "Menu 1", Link = "#"},
-        new MenuItem { Icon = "bi bi-alarm",  MenuTitle = "Menu 2", Link = "#"},
-        new MenuItem { Icon = "bi bi-xbox",  MenuTitle = "Menu 3", Link = "#"},
+        new MenuItem { Icon = "bi bi-building",  MenuTitle = "Booking", Link = "#"},
+        new MenuItem { Icon = "bi bi-alarm",  MenuTitle = "Chat", Link = "#"},
+        new MenuItem { Icon = "bi bi-xbox",  MenuTitle = "Menu 3", Link = ""},
         new MenuItem { Icon = "bi bi-youtube",  MenuTitle = "Menu 4", Link = "#"},
-        new MenuItem { Icon = "bi bi-wifi",  MenuTitle = "Menu 5", Link = "#"},
-        new MenuItem { Icon = "bi bi-vimeo",  MenuTitle = "Menu 6", Link = "#"}
+        new MenuItem { Icon = "bi bi-wifi",  MenuTitle = "Document", Link = "./Home/DocumentArchive"},
+        new MenuItem { Icon = "bi bi-vimeo",  MenuTitle = "Contact", Link = "./Home/Contact"}
```

Comment:
Du skal gerne ikke benytte hardcoded link urls som her, men istedet `Url.Action()`
Dokumentation kan findes her: https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.urlhelperextensions?view=aspnetcore-6.0

Guide: https://nimblegecko.com/how-to-link-to-controller-actions-from-views-in-asp-net-core/
```suggestion
        new MenuItem { Icon = "bi bi-wifi",  MenuTitle = "Document", Link = "@(Url.Action("DocumentArchive", "Home"))"},
        new MenuItem { Icon = "bi bi-vimeo",  MenuTitle = "Contact", Link = "@(Url.Action("Contact", "Home"))"}
```

Husk lige at tjek at dette compiler. Jeg har bare skrevet det gennem GitHub